lib/ansible/plugins/loader.py

```diff
@@ -145,16 +145,17 @@ def record_deprecation(self, name, deprecation, collection_name):
         # If both removal_date and removal_version are specified, use removal_date
         if removal_date is not None:
             removal_version = None
-        if not warning_text:
-            if removal_date:
-                warning_text = '{0} has been deprecated and will be removed in a release of {2} after {1}'.format(
-                    name, removal_date, collection_name)
-            elif removal_version:
-                warning_text = '{0} has been deprecated and will be removed in version {1} of {2}'.format(
-                    name, removal_version, collection_name)
-            else:
-                warning_text = '{0} has been deprecated and will be removed in a future release of {2}'.format(
-                    name, collection_name)
+        if removal_date:
+            warning_text_res = '{0} has been deprecated and will be removed in a release of {2} after {1}'.format(
+                name, removal_date, collection_name)
+        elif removal_version:
+            warning_text_res = '{0} has been deprecated and will be removed in version {1} of {2}'.format(
+                name, removal_version, collection_name)
+        else:
+            warning_text_res = '{0} has been deprecated and will be removed in a future release of {2}'.format(
```

Comment:
```suggestion
            warning_text_res = '{0} has been deprecated and will be removed in a future release of {1}'.format(
```

Reply:
Small fix. Else, this results in a tuple out of range error. :)

Reply:
Thanks for spotting this! :)